tests/Tokenizer/Transformer/TransformersTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\tests\Tokenizer\Transformer;
+
+use PhpCsFixer\Tokenizer\Transformers;
+
+/**
+ * @internal
+ *
+ * @author SpacePossum
+ */
+final class TransformersTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @param \ReflectionClass $transformerRef
+     *
+     * @dataProvider provideReflectedTransformers
+     */
+    public function testTransformersAreFinal(\ReflectionClass $transformerRef)
```

Comment:
but it's inside msg of assertion, which may be there or not for new assertions in given test.
at the same time, test indicator should be meaningful, currently all of them are just showing `ReflectionClass`